covid_xprize/validation/scenario_generator.ipynb

```diff
@@ -7,6 +7,18 @@
     "# Copyright 2020 (c) Cognizant Digital Business, Evolutionary AI. All rights reserved. Issued under the Apache 2.0 License."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add the covid_xprize directory to the system path.\n",
+    "import sys\n",
+    "import os\n",
+    "sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(''))))"
```

Comment:
That should not be necessary if you set `PYTHONPATH` to the root folder in the terminal you use to start jupyter lab. For instance with:
```
 export PYTHONPATH=`pwd`
```